File: mobile/src/main/javaBackup/com/planit/home/planit007/data/Contracts/MemberContract.java
package planit007.planit.home.planit007.data.Contracts;

import android.provider.BaseColumns;

/**
 * Created by Home on 2017-08-02.
 */

public class MemberContract {

    public class MemberEntry implements BaseColumns {

        public static final String TABLE_NAME = "members";

        public static final String COLUMN_NAME = "name";

        public static final String COLUMN_GROUP_ID = "groupBelongedTo";

        public static final String COLUMN_TIMESLOT_MEMBER_AVAILABILITY = "tMemberAvailability";

        public static final String COLUMN_MEMBER_QUALIFICATIONS = "tMemberQualifications";

        public static final String COLUMN_MEMBER_PREFERENCES = "tMemberPreferences";

        public static final String COLUMN_MEMBER_AVAILABILITY = "tMemberAvailability";
    }
}

File: mobile/src/main/kotlin/com/planit/mobile/data/Contracts/MemberContract.kt
package com.planit.mobile.data.Contracts

import android.provider.BaseColumns

/**
 * Created by Home on 2017-08-02.
 */

class MemberContract {

    class MemberEntry : BaseColumns {
        companion object {

            val _ID = "_id"

            val TABLE_NAME = "members"

            val COLUMN_NAME = "name"

            val COLUMN_GROUP_ID = "groupBelongedTo"

            val COLUMN_TIMESLOT_MEMBER_AVAILABILITY = "tMemberAvailability"

            val COLUMN_MEMBER_QUALIFICATIONS = "tMemberQualifications"

            val COLUMN_MEMBER_PREFERENCES = "tMemberPreferences"

            val COLUMN_MEMBER_AVAILABILITY = "tMemberAvailability"
        }
    }
}
